src-tauri/src/spito_helper.rs

```diff
@@ -0,0 +1,51 @@
+use std::process::{Command};
+use dbus_crossroads::Crossroads;
+use dbus::blocking::Connection;
+use tauri::{AppHandle, Manager};
+
+#[tauri::command]
+pub fn start_spito_cli(ruleset: &str, rule: &str) -> Result<(), String> {
+    Command::new("spito")
+        .arg("check")
+        .arg("--gui-child-mode")
+        .arg(ruleset)
+        .arg(rule)
+        .spawn().map_err(|err| err.to_string())?;
+    Ok(())
+}
+
+const DBUS_ID: &str = "org.avorty.spito.gui";
+const DBUS_OBJECT_PATH: &str = "/org/avorty/spito/gui";
+
+#[derive(Clone, serde::Serialize)]
+struct DBusEchoPayload {
+    mess_type: String,
```

Comment:
Make it camel case when serialized (you can use serde decorator for this) and consider using full name

Reply:
RustRover says it should be snake case